### PR TITLE
Add quota service permissions to the `installer policy`

### PR DESCRIPTION
**Pull request by oriAdler**
This MR will enable CS to run preflight quota checks.

Related: [SDA-5429](https://issues.redhat.com/browse/SDA-5429)

### CODE REVIEW

**Comment by oriAdler** [comment]
@zgalor @pvasant PTAL

**Comment by pvasant** [comment]
@jharrington22  @vkareh @arendej  PTAL

**Comment by vkareh** [comment]
This looks good to me.

I would like @arendej's opinion before merging

File: templates/policies/sts_installer_permission_policy.json [hunk 1]
```diff
@@ -169,6 +169,8 @@
                 "s3:PutObject",
                 "s3:PutObjectAcl",
                 "s3:PutObjectTagging",
+                "servicequotas:GetServiceQuota",
+                "servicequotas:ListAWSDefaultServiceQuotas",
```
Comment (jharrington22) [inline templates/policies/sts_installer_permission_policy.json:173]: @oriAdler do you only need these two? We were initially requesting these as part of the OSD service definition SCP policy

```
            "Action": [
                "servicequotas:ListServices",
                "servicequotas:GetRequestedServiceQuotaChange",
                "servicequotas:GetServiceQuota",
                "servicequotas:RequestServiceQuotaIncrease",
                "servicequotas:ListServiceQuotas"
            ],
```

Comment (pvasant) [reply templates/policies/sts_installer_permission_policy.json:173]: @jharrington22  this PR is primarily focused on the preflight requirement. So we kept it as just listing and read-only permissions. Why do we need   "servicequotas:ListServices" though 
https://issues.redhat.com/browse/SDE-1545

**Comment by arendej** [comment]
LGTM

File: templates/policies/sts_installer_permission_policy.json (hunk 1 above)
Comment (oriAdler) [reply templates/policies/sts_installer_permission_policy.json:173]: I'm currently working on adding the [`EIPs` preflight check](https://issues.redhat.com/browse/SDA-5288) and using the next API call:
`client.GetServiceQuota(ctx, QuotaCode, ServiceCode)`

So this permission `"servicequotas:GetServiceQuota"` is sufficient at the moment.

I assume that for `SDA-4852 Internet Gateway Exceeded`, `SDA-5015 - NAT Gateway Exceeded`, 
`SDA 5043 - Too many buckets`, `SDA-5287 -  ec2 1216C47A quota` we'll use the same API call
to get the resource limit on the account.

Comment (jharrington22) [reply templates/policies/sts_installer_permission_policy.json:173]: Ok this make sense, I thought you were listing and checking the same way the CLI does it but you have a defined list of service codes. We have the RequestServiceQuotaIncrease for the AWS account operator.

**Review by jharrington22 — APPROVED** [review]

**Pull request merged**